### ISSUE TITLE
Windows Installation

### ISSUE THREAD

**Issue by SivuyileNzimeni** [quarto-dev/quarto-cli#832]
I have installed Quarto on MacOs without issue. However, installation on Windows requires a person to add Quarto path  Environment Variables.  It may be worthwhile to add a note to the Windows installation guide to assist new users.

**Comment by eitsupi** [comment]
Maybe duplicate of #804

**Comment by jjallaire** [comment]
In theory we should be adding Quarto to the PATH (although it sometimes requires a restart of your PowerShell or IDE to be visible). @dragonstyle What do you think?

**Comment by dragonstyle** [comment]
I just confirmed that we are adding the Quarto installation directory to the path (either user or system depending upon install type). I haven't been able to find a way to signal existing command prompts, powershell windows, or vs code instances to reload their environment.

We are conditionally including text to let the user know about this (e.g. instructing to restart these apps prior to using Quarto), but the condition as it was written meant it wasn't showing for system installations. I have updated the logic and tried to improve the message as well- the message should so for all installations.
